cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Committee.hs

```diff
@@ -71,7 +72,13 @@ data GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs era
   = GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs
   { eon :: !(ConwayEraOnwards era)
   , vkeyColdKeySource :: !(VerificationKeySource CommitteeColdKey)
-  , anchor :: !(Maybe (L.Anchor (L.EraCrypto (ShelleyLedgerEra era))))
+  , anchor
+      :: !( Maybe
```

Comment:
I'm really not a fan of  `PotentiallyCheckedAnchor` because this is an input argument submitted by the user and _they are deciding_ whether or not to check it.  IIRC @smelc also was leaning towards having two separate arguments for this. We don't need to deal with this in this PR but the name is definitely misleading. Having a resulting type that indicates if the hash has been checked or not would be more appropriate vs `PotentiallyCheckedAnchor` which is telling us if we still have to check the hash or not.

Reply:
Do you mean the name of the parameter the user writes should be called differently? Or that the code should be structured different? The reason the type is wrapped in `PotentiallyCheckedAnchor` is that the option to check the hash only makes sense if there is a hash, and it is optional, so this way we can disallow providing the check requirement and not the hash